pkg/imrpc/instance.proto

```diff
@@ -0,0 +1,92 @@
+syntax="proto3";
+
+package imrpc;
+
+import "google/protobuf/empty.proto";
+import "imcommon.proto";
+
+service InstanceService {
+    rpc InstanceCreate(InstanceCreateRequest) returns (InstanceResponse) {}
+	rpc InstanceDelete(InstanceDeleteRequest) returns (InstanceResponse) {}
+	rpc InstanceGet(InstanceGetRequest) returns (InstanceResponse) {}
+	rpc InstanceList(google.protobuf.Empty) returns (InstanceListResponse) {}
+	rpc InstanceLog(InstanceLogRequest) returns (stream LogResponse) {}
+	rpc InstanceWatch(google.protobuf.Empty) returns (stream google.protobuf.Empty) {}
+	rpc InstanceReplace(InstanceReplaceRequest) returns (InstanceResponse) {}
+
+	rpc VersionGet(google.protobuf.Empty) returns(VersionResponse);
+}
+
+message ProcessSpecific {
```

Comment:
`ProcessSpecific` seems the word is not finished. How about `ProcessInstace` or `ProcessInstanceSpec`?

Reply:
`ProcessSpecific` means they are **specific** for `Process`.
`ProcessInstace` or `ProcessInstanceSpec` sounds not suitable.

Reply:
I see. How about `ProcessSpec`? `ProcessSpecific` seems an adjective, which is not quite suitable for a class name. WDYT?

Reply:
I'm fine with it.

Reply:
Addressed. Thank you.